ci_framework/roles/edpm_deploy_baremetal/tasks/main.yml

```diff
@@ -0,0 +1,95 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Set install_yamls Makefile environment variables
+  vars:
+    operators_build_output: "{{ (cifmw_operator_build_output | default({'operators':{}})).operators }}"
+  ansible.builtin.set_fact:
+    cifmw_edpm_deploy_baremetal_common_env:
+      PATH: "{{ cifmw_path }}"
+    cifmw_edpm_deploy_baremetal_make_openstack_env: |
+      {% if cifmw_operator_build_meta_name is defined and cifmw_operator_build_meta_name in operators_build_output %}
+      OPENSTACK_IMG: {{ operators_build_output[cifmw_operator_build_meta_name].image_catalog }}
+      OPENSTACK_BRANCH: ""
+      GIT_CLONE_OPTS: "-l"
+      OPENSTACK_REPO: {{ operators_build_output[cifmw_operator_build_meta_name].git_src_dir }}
+      {% endif %}
+    cifmw_edpm_deploy_baremetal_operators_build_output: "{{ operators_build_output }}"
+
+- name: Prepare storage in CRC
+  vars:
+    make_crc_storage_env: "{{ cifmw_edpm_deploy_baremetal_common_env }}"
+    make_crc_storage_dryrun: "{{ cifmw_edpm_deploy_baremetal_dry_run }}"
+  when: not cifmw_edpm_deploy_baremetal_skip_crc_storage_creation | bool
+  ansible.builtin.include_role:
+    name: 'install_yamls_makes'
+    tasks_from: 'make_crc_storage'
+
+- name: Install OpenStack operator
+  vars:
+    make_openstack_env: "{{ cifmw_edpm_deploy_baremetal_common_env |
+      combine(cifmw_edpm_deploy_baremetal_make_openstack_env | from_yaml)}}"
+    make_openstack_dryrun: "{{ cifmw_edpm_deploy_baremetal_dry_run }}"
+  ansible.builtin.include_role:
+    name: 'install_yamls_makes'
+    tasks_from: 'make_edpm_compute_baremetal'
+
+- name: Wait for OpenStack Provision Server pod to be created
+  environment:
+    PATH: "{{ cifmw_path }}"
+  ansible.builtin.command: oc get po -l osp-provisionserver/name=edpm-compute-provisionserver -n {{ cifmw_install_yamls_defaults['NAMESPACE'] }} -o name
+  register: cifmw_edpm_deploy_baremetal_provisionserver_pod_output
+  retries: "{{ cifmw_edpm_deploy_baremetal_wait_provisionserver_retries }}"
+  delay: 10
+  until: cifmw_edpm_deploy_baremetal_provisionserver_pod_output.stdout != ''
+
+- name: Wait for OpenStack Provision Server deployment to be available
+  environment:
+    PATH: "{{ cifmw_path }}"
+  ansible.builtin.shell: oc wait deployment edpm-compute-provisionserver-openstackprovisionserver -n {{ cifmw_install_yamls_defaults['NAMESPACE'] }} --for condition=Available --timeout={{ cifmw_edpm_deploy_baremetal_wait_provisionserver_timeout }}m
```

Comment:
@cjeanner we will need to standardize time unit. Anyway, putting 'sec' or 'mins' as part of the variable name helps a lot.

Reply:
@pablintino added _mins to the variable names.